app/controllers/api/api_controller.rb

```diff
@@ -25,23 +25,25 @@ def dashboard
     end
 
     begin
-      @countries_data = {}
-      countries_calculations = {}
+      @countries_data = Array.new
```

Comment:
I had a comment in here before about this being gross, which was lost in a refactor. This is probably still on the gross side, and I also need to remember to fix the tests for it.